code/game/gamemodes/trifecta/trifecta.dm

```diff
@@ -0,0 +1,115 @@
+#define TOT_COST 5
+#define VAMP_COST 10
+#define CLING_COST 10
+
+
+/datum/game_mode/trifecta
+	name = "Trifecta"
+	config_tag = "trifecta"
+	protected_jobs = list("Security Officer", "Warden", "Detective", "Head of Security", "Captain", "Blueshield", "Nanotrasen Representative", "Magistrate", "Chaplain", "Internal Affairs Agent", "Nanotrasen Navy Officer", "Special Operations Officer", "Solar Federation General")
```

Comment:
This and line 63 prevents you from rolling traitor chaplain, I believe

Reply:
This also prevents changeling chaplain I think, which I dont think is intended either

Reply:
Will fix it

Reply:
Fixed